x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -487,17 +594,33 @@ export const entryHasNonEcsType = (
 export const defaultEndpointExceptionItems = (
   listId: string,
   ruleName: string,
-  alertEcsData: Ecs
+  alertEcsData: FlattenType<Ecs>
 ): ExceptionsBuilderExceptionItem[] => {
-  const { file, event: alertEvent } = alertEcsData;
+  const { file, process, Ransomware, event: alertEvent } = alertEcsData;
```

Comment:
I don't think `file` is used here.

Reply:
It's used [here](https://github.com/elastic/kibana/pull/89974/files/026b4481c5fe3985a5d14d7d97f3ab573c8b329d#diff-6808c52ab81432d126e55b4a4d4ff277c9fb83276e3f9170ca8f0ae1ce922efcR621)